src/user/announcements.rst

```diff
@@ -8,20 +8,31 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2023
 ----
 
+2023-07-12: End-of-life for CentOS 6
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As you may be aware, we have delayed the deprecation of our CentOS 6 build
+system the ``linux64`` platform several times. We have now set a formal deprecation
+date to be June 30, 2024. This date matches the
+`end of extended life-cycle support <https://endoflife.software/operating-systems/linux/red-hat-enterprise-linux-rhel>`_
+from RedHat for RHEL 6. After this date, we build packages against
+CentoOS 7 by default for ``linux64``.
```

Comment:
```suggestion
CentosOS 7 by default for ``linux64``.
```

Reply:
fixed!

Reply:
Lol and my fix had a typo 🤣

Thanks Matt! 🙏